docs/video-miners/terminology.md

```diff
@@ -3,55 +3,66 @@ title: Terminology
 sidebar_position: 6
 ---
 
-## Transcoding
+## Adaptive Bitrate (ABR) ladder
 
-Transcoding is the process of taking an input source video in one format andbitrate, and converting it to many output rendition videos with different formats and bitrates such that the video can be played on the majority of the
-devices in the world at any internet connection speed. 
+An Internet Video Streaming industry standard term, ABR streaming is known as "Adaptive Bitrate Streaming". The Adaptive Bitrate (ABR) ladder can be considered metaphorically and refers to "the array of segments of different quality and resolution which are available from the streaming server. If the bitrate rises, the logic in a video player can select a larger size file with better quality to play, thus resembling the "climbing up of the ladder".
+
+<!--- https://developer.att.com/video-optimizer/docs/best-practices/adaptive-bitrate-video-streaming -->
+
+## Capacity
+
+Capacity refers to the total transcoding that an orchestrator or transcoder can provide in an acceptable time frame (i.e. real-time). At the moment, orchestrators and transcoders can set their capacities based on the maximum number of sessions that they can support. For an orchestrator, this maximum session count refers to the maximum number of simultaneous jobs they can receive from broadcasters. For a transcoder, this maximum session count refers to the maximum number of simultaneous transcoding tasks they can receive from broadcasters.
 
-Additional details on transcoding can be found in
-[this blog post](https://livepeer.com/blog/intro-to-transcoding).
+## Concurrent Sessions
+
+***See*** the definition of [Sessions](video-miners/terminology#session)
```

Comment:
`See` is capitalized here but below it's not, should we stay consistent?

Reply:
yes, it should be capitalized.